src/inventoryWindows.ts

```
@@ -482,7 +482,12 @@ const openWindow = (type: string | undefined, title: string | any = undefined) =
       oldOnInventoryEvent(type, containing, windowIndex, inventoryIndex, item)
     }
   }
+  let jeiClickInProgress = false
```

ESLint

[error] 'jeiClickInProgress' is never reassigned. Use 'const' instead.

(prefer-const)